services/stats/stats.go

```diff
@@ -47,17 +47,23 @@ func init() {
 	enableCPUStats = config.GetBool("RuntimeStats.enableCPUStats", true)
 	enableMemStats = config.GetBool("RuntimeStats.enabledMemStats", true)
 	enableGCStats = config.GetBool("RuntimeStats.enableGCStats", true)
+}
+
+//CreateStatsClient creates a new statsd client
+func CreateStatsClient() {
```

Comment:
Let's call it something like initialize or setup?

Reply:
Done